src/main/scala/chisel3/compatibility.scala

```diff
@@ -554,43 +661,12 @@ package object Chisel {
       * }
       * }}}
       */
+    @deprecated("Chisel compatibility mode is deprecated. Use the chisel3 package instead.", "Chisel 3.6")
     def apply[T <: Bits](nodeType: T, n: Int): List[T] = {
       require(nodeType.isInstanceOf[UInt], "Only UInt supported for enums")
       require(!nodeType.widthKnown, "Bit width may no longer be specified for enums")
       apply(n).asInstanceOf[List[T]]
     }
-
-    /** An old Enum API that returns a map of symbols to UInts.
-      *
-      * Unlike the new list-based Enum, which can be unpacked into vals that the compiler
-      * understands and can check, map accesses can't be compile-time checked and typos may not be
-      * caught until runtime.
-      *
-      * Despite being deprecated, this is not to be removed from the compatibility layer API.
-      * Deprecation is only to nag users to do something safer.
-      */
-    @deprecated("Use list-based Enum", "not soon enough")
-    def apply[T <: Bits](nodeType: T, l: Symbol*): Map[Symbol, T] = {
-      require(nodeType.isInstanceOf[UInt], "Only UInt supported for enums")
-      require(!nodeType.widthKnown, "Bit width may no longer be specified for enums")
-      (l.zip(createValues(l.length))).toMap.asInstanceOf[Map[Symbol, T]]
-    }
-
-    /** An old Enum API that returns a map of symbols to UInts.
-      *
-      * Unlike the new list-based Enum, which can be unpacked into vals that the compiler
-      * understands and can check, map accesses can't be compile-time checked and typos may not be
-      * caught until runtime.
-      *
-      * Despite being deprecated, this is not to be removed from the compatibility layer API.
-      * Deprecation is only to nag users to do something safer.
-      */
-    @deprecated("Use list-based Enum", "not soon enough")
-    def apply[T <: Bits](nodeType: T, l: List[Symbol]): Map[Symbol, T] = {
-      require(nodeType.isInstanceOf[UInt], "Only UInt supported for enums")
-      require(!nodeType.widthKnown, "Bit width may no longer be specified for enums")
-      (l.zip(createValues(l.length))).toMap.asInstanceOf[Map[Symbol, T]]
-    }
   }
```

Comment:
Even though these have been deprecated, is there any value in deleting them vs. just leaving them until we delete all of this file?

Reply:
putting them back